mobile/voice_journal_app/lib/RecordingPage.dart

```diff
@@ -170,8 +207,8 @@ class _RecordingPageState extends State<RecordingPage>{
         child: FloatingActionButton(
           onPressed: _changeState,
           tooltip: 'Record',
-          backgroundColor: Colors.red[200], //button background colour
-          splashColor: Colors.red[500],     //button click animation
+          backgroundColor: AppColors.accentColor, //button background colour
+          splashColor: Colors.red[200],     //button click animation
           child: rcrdIcon,
```

Comment:
Instead of keeping a widget as a variable and updating it based on what we need, let's keep the widgets within the `build` function and show one or another based on some condition. For example:
```
child: recording ? const Icon(Icons.mic_off_outlined) : const Icon(Icons.pause),
```
Here we set one icon or another based on the recording variable that gets updated thru the state.